modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,537 @@
+
+In particular, this includes ensuring that formatting is not directly removed from elements with `+contenteditable=”false”+` attributes when merging duplicate or similar formatting between parent, sibling and child elements.
+
+=== Spaces could be incorrectly added to `urlinput` dialog components in certain cases
+
+The handling of spaces inserted into the xref:dialog-components/#urlinput[UrlInput] dialog component, specifically its implementation in the xref:link[Link Dialog], was inconsistent.
+
+It allowed spaces to be entered when it was empty and the text input contained text, but did not allow spaces to be entered when both fields were empty.
+
+In {productname} 6.2, the xref:dialog-components/#urlinput[UrlInput] dialog component no longer allows leading or trailing spaces. Only spaces within other text content are now allowed.
+
+
+=== The text patterns logic threw an error when there were fragmented text nodes in a paragraph
+
+Previously, pressing *Enter* or *Return* to trigger an inline replacement text pattern could, in certain circumstances, throw an error to the console.
+
+[listing]
+----
+> tinymce.min.js:9 Uncaught DOMException: Failed to execute 'setEnd' on 'Range'
+----
+
+For this update the text pattern logic was improved to handle these cases and these console errors are no longer thrown.
+
+=== Dragging a `contentEditable=false` element towards a document’s edge did not cause scrolling
+
+Previously, when an element with a `contenteditable=false` attribute was dragged towards the document’s edge, the drag did not cause scrolling. This meant such elements could only be dropped within the current viewport.
+
+When dragging other elements (such as an image), as the dragged element approaches the edge of the document, {productname} starts scrolling, allowing these other elements to be dropped anywhere in the document.
+
+With this update, dragging an element with a `contenteditable=false` attribute towards the document edge triggers scrolling, allowing for dropping the element anywhere in the document.
+
+In {productname} 6.2, dragging elements with `contentEditable=false` attributes now results in the same behavior as dragging other draggable elements.
+
+
+=== Parsing large documents no longer throws a Maximum call stack size exceeded exception
+
+Previously, parsing very large documents would throw an error to the console that they had exceeded the Maximum call stack size. 
+
+For example:
+
+[listing]
+----
+tinymce.min.js:9 Uncaught RangeError: Maximum call stack size exceeded
+    at tinymce.min.js:9:144118
+    at zp (tinymce.min.js:9:143461)
+    at zp (tinymce.min.js:9:143480)
+    at zp (tinymce.min.js:9:143507)
+    at zp (tinymce.min.js:9:143507)
+----
+
+the {productname} 6.2 parsing logic was updated and {productname} can now handle large documents without throwing this error.
```

Comment:
Should this be a capital? It looks like a new sentence to me 🤷

Reply:
(discussion): typo